core/domain/email_manager.py

```diff
@@ -77,6 +77,46 @@ def log_new_error(*args, **kwargs):
         'subject': _PLACEHOLDER_SUBJECT,
         'html_body': _PLACEHOLDER_HTML_BODY,
     })
+
+_EDITOR_ROLE_EMAIL_SUBJECT = (
```

Comment:
Actually, you might want to get rid of these as separate constants altogether, and move them into the main body of the function where you interpolate them. The reason for this is that it's nice to have the interpolated string template near its arguments, so you can see what the arguments map to. Otherwise you have to do a fair bit of cross-referencing.